bot/exts/utilities/githubinfo.py

```diff
@@ -192,7 +195,7 @@ async def on_message(self, message: discord.Message) -> None:
 
             log.trace(f"Found {issues = }")
             # Remove duplicates
-            issues = set(issues)
+            issues = unique_everseen(issues)
```

Comment:
```suggestion
            issues = list(unique_everseen(issues))
```
The function returns an iterator which does not have a length that is checked below